users/views.py

```diff
@@ -444,18 +444,39 @@ def username_from_username_part(request):
 
 def balance_from_username(request):
 
-    ## NEED permissions
+    ## Check permissions
+
+    # User is authentified, if not the he can't access the view
+    operator = request.user
+
+    # try:
+    #     shop_name = request.GET.get('shop_name')
+    #     shop = Shop.objects.get(name = shop_name)
+    #     module = OperatorSaleModule.objects.get(shop = shop)
+    # except KeyError:
+    #         raise Http404
+    # except ObjectDoesNotExist:
+    #     raise Http404
+
+    # If deactivate
+    # if module.state is False:
+    #     raise Http404
+
+    if operator.has_perm('modules.use_operatorsalemodule'):
```

Comment:
Does this function could be used elsewhere ?
If so, we should change the permission to something like "get_users_balance", don't know hehe

Reply:
Yeah, i agree 👍